ml/Database.h

```diff
@@ -73,6 +74,16 @@ class Statement {
         return bindValue(Pos, T) + bind(Pos + 1, Args...);
     }
 
+    /*template<const int, typename ...ArgTypes>
+    size_t bind(size_t Pos, const int T, ArgTypes ...Args) {
+        return bindValue(Pos, T) + bind(Pos + 1, Args...);
+    }*/
+
+    /*template<float, typename ...ArgTypes>
+    size_t bind(size_t Pos, float T, ArgTypes ...Args) {
+        return bindValue(Pos, T) + bind(Pos + 1, Args...);
+    }*/
+
```

Comment:
Commenting out existing code that we want to keep can be a double-edge sword, because it might hide compilation errors caused by newly added code.

Reply:
This is interim, you are right, I will complete this when I clarify overloaded functions based on both int and double.